core/controllers/question_editor.py

```diff
@@ -76,16 +76,19 @@ def post(self, question_id, skill_id):
         if not constants.ENABLE_NEW_STRUCTURE_EDITORS:
             raise self.PageNotFoundException
 
-        skill_domain.Skill.require_valid_skill_id(skill_id)
-        skill = skill_services.get_skill_by_id(skill_id, strict=False)
-        if skill is None:
-            raise self.PageNotFoundException(
-                'The skill with the given id doesn\'t exist.')
-
-        # TODO(vinitamurthi): Replace DEFAULT_SKILL_DIFFICULTY
-        # with a value passed from the frontend.
-        question_services.create_new_question_skill_link(
-            question_id, skill_id, constants.DEFAULT_SKILL_DIFFICULTY)
+        for single_skill_id in skill_id.split(','):
```

Comment:
I understand why you need to use comma-separated string format for GET, but shouldn't you be using a list of strings for POST (in the payload)? That would be more natural and is well-supported by JSON.

Reply:
DOne.